homeassistant/components/tasmota/sensor.py

```diff
@@ -80,6 +81,7 @@
     },
     hc.SENSOR_CCT: {
         ICON: "mdi:temperature-kelvin",
+        DEVICE_CLASS: SensorDeviceClass.TEMPERATURE,
```

Comment:
Is this correct?

Reply:
This is measuring color temperature; I think it's weird to classify that as a temperature?

Reply:
```suggestion
```